zerver/views/user_settings.py

```diff
@@ -55,13 +55,12 @@ def confirm_email_change(request: HttpRequest, confirmation_key: str) -> HttpRes
 
 @human_users_only
 @has_request_variables
-def json_change_ui_settings(request, user_profile,
-                            autoscroll_forever=REQ(validator=check_bool,
-                                                   default=None),
-                            default_desktop_notifications=REQ(validator=check_bool,
-                                                              default=None)):
-    # type: (HttpRequest, UserProfile, Optional[bool], Optional[bool]) -> HttpResponse
-
+def json_change_ui_settings(request: HttpRequest, user_profile: UserProfile,
+                            autoscroll_forever: Optional[bool]=REQ(validator=check_bool,
+                                                                   default=None),
+                            default_desktop_notifications: Optional[bool]=REQ(validator=check_bool,
+                                                                              default=None)
+                            ) -> HttpResponse:
```

Comment:
Looking back here, I think [this](https://github.com/zulip/zulip/pull/7653/files#diff-2a3f35796feeb6b0ded8f5b2eb2a58f0R172) style again would look better in this case.

Sorry for the back and forth, but it's takes a bit of experimenting to find out what looks best.